server/commands.py

```diff
@@ -6157,6 +6156,47 @@ def ooc_cmd_zone_global(client: ClientManager.Client, arg: str):
                         .format(client.area.id, client.displayname),
                         pred=lambda c: not c.muted_global)
 
+def ooc_cmd_zone_lights(client: ClientManager.Client, arg: str):
+    """ (STAFF ONLY)
+    Toggles lights on or off in the background for every area in a zone. If turned off,
+    the background will change to the server's blackout background. If turned on,
+    the background will revert to the background before the blackout one.
+    If an area already has the requested light status, the area is left alone.
```

Comment:
Add to your description that you won't change the light status of areas that either have their background locked, or that by construction have no lights